main.cpp

```diff
@@ -285,9 +320,13 @@ int main(int argc, char** argv)
     /// Assign Pareto optimal values
     if(prob == 5) {
         cout << "\n% Optimal set cannot be analytically derived for CODeM5" << endl;
+        if (writeJsonFile){
+            jsonOutput["status"] = "Optimal set cannot be analytically derived for CODeM5";
```

Comment:
Should this status be used by the program that reads the json file? Maybe it should be a Boolean that states whether the optimal set can be derived?

Reply:
It is really not used. I will delete all references to status.